src/bindings/python/tests/test_transformations/test_offline_api.py

```diff
@@ -203,12 +203,12 @@ def test_compress_model_transformation():
     node_constant = ov.opset8.constant(np.array([[0.0, 0.1, -0.1], [-2.5, 2.5, 3.0]], dtype=np.float32))
     node_ceil = ov.opset8.ceiling(node_constant)
     model = Model(node_ceil, [], "TestModel")
-    elem_type = model.get_ordered_ops()[0].get_element_type().get_type_name()
```

Comment:
will we deprecated old method?

Reply:
We have `get_type_name()` for some classes... I think we can deprecate all these methods in separate PR